src/app/App.js

```diff
@@ -41,9 +41,11 @@ export const App = ({options}) => {
    });
 
    // start the game
-   document.querySelector("#startGameButton").addEventListener("click", () => {
-      const userName = document.querySelector('#enterYourNameInput').value;
-      renderQuizPage(SELECTED_MODE, userName, options.quizMaxTime)
-   });
+   document.querySelector('#enterYourNameInput').addEventListener("change", () =>{
+      document.querySelector("#startGameButton").addEventListener("click", () => {
+         const userName = document.querySelector('#enterYourNameInput').value;
+         renderQuizPage(SELECTED_MODE, userName, options.quizMaxTime)
+      });
+   })
```

Comment:
I don't think this is a good approach. By this way you will be adding new event listener to #startGameButton every time you change something on name input. If somebody will write 1000 letters there -> then there will be 1000 listeners that will render quiz page. Not good :) 

Better approach is look at this:
https://stackoverflow.com/questions/7067005/disable-button-whenever-a-text-field-is-empty-dynamically

- make "PLAY" button disabled by default.
- add "onkeyup" (not sure about exact name) event listener to the "name input" that will execute something like this: 
```
function stoppedTyping(){
        if(this.value.length > 0) { 
            document.getElementById('start_button').disabled = false; 
        } else { 
            document.getElementById('start_button').disabled = true;
        }
    }
```